transport/redis/internal.thrift

```diff
@@ -0,0 +1,14 @@
+struct RPC {
```

Comment:
I didn't want to call it Request since it holds more than just the transport.Request details. Also I feel like RPCRequest is a bit redundant?